### ISSUE TITLE
SQLite Error 14: Unable to open database file

### ISSUE THREAD

**Issue by algolee** [porrey/Virtual-ZPL-Printer#45]
After running the installer for version **3.0.1** and attempting to run the application, the following error appears.

![Screenshot 2024-01-11 100649](https://github.com/porrey/Virtual-ZPL-Printer/assets/12646250/648b39ee-7eda-4388-9e09-24af23c1157b)

Dismissing the error (by clicking the OK button), opens the app. However, adding label settings is not possible. This error renders the app unusable.

![Screenshot 2024-01-11 101107](https://github.com/porrey/Virtual-ZPL-Printer/assets/12646250/8cdcdc9c-b81d-4b53-9fdb-6b34f7607ae3)

**Comment by algolee** [comment]
the "db.sqlite" does not exist in C:\Users\xyz\Documents\Virtual ZPL Printer\Database

**Comment by porrey** [comment]
Sorry you're having this issue. 

The database should auto-create if it does not exist. I deleted my database and then started the app and it successfully created a new database file.

I was able to reproduce the issue by "corrupting" my database (I deleted the tables). After deleting the database the application recreated it.

Try closing the app, rename the **`Virtual ZPL Printer`** folder to something like **`Virtual ZPL Printer Backup`** and relaunch the application. The setup should launch and rebuild the folder.
